server/routers/api-router.js

```diff
@@ -1,242 +1,240 @@
-let express = require("express");
-const { allowDevAllOrigin, getSettings, setting } = require("../util-server");
-const { R } = require("redbean-node");
-const server = require("../server");
-const apicache = require("../modules/apicache");
-const Monitor = require("../model/monitor");
-const dayjs = require("dayjs");
-const { UP, flipStatus, debug } = require("../../src/util");
+let express = require('express');
+const { allowDevAllOrigin, getSettings, setting } = require('../util-server');
+const { R } = require('redbean-node');
+const server = require('../server');
+const apicache = require('../modules/apicache');
+const Monitor = require('../model/monitor');
+const dayjs = require('dayjs');
+const { UP, flipStatus, debug, DOWN } = require('../../src/util');
 let router = express.Router();
 
 let cache = apicache.middleware;
 let io = server.io;
 
-router.get("/api/entry-page", async (_, response) => {
-    allowDevAllOrigin(response);
-    response.json(server.entryPage);
+router.get('/api/entry-page', async (_, response) => {
+	allowDevAllOrigin(response);
```

Comment:
It becomes a real tab character. Feel like you are not using our repo's eslint config.